src/Processors/QtiV2/In/Interactions/TextEntryInteractionMapper.php

```diff
@@ -25,6 +47,35 @@ public function getQuestionType()
         return $closetext;
     }
 
+    public function getOption($key, $defaultValue = null) {
+        $value = $defaultValue;
+        if (isset($this->options[$key])) {
+            $value = $this->options[$key];
+        }
+
+        return $value;
+    }
+
+    public function isUsingExpectedLengthAsMaxCharLimit() {
+        return (bool)$this->getOption('expected_length_as_max_char_limit', false);
+    }
+
+    public function isUsingExpectedLengthAsResponseWidth() {
+        return (int)$this->getOption('expected_length_as_response_width', true);
+    }
+
+    public function setOption($key, $value) {
```

Comment:
Lets limit the visibility scope here also.

Reply:
I've converted this into a `setOptions(array $options)` method. This actually should be public - it's there so these options can be injected in via a config object.